sdk/samples/iot/paho_iot_hub_twin_cbor_sample.c

```diff
@@ -460,37 +461,56 @@ static bool parse_desired_device_count_property(
   bool property_found = false;
   *out_parsed_device_count = 0;
 
+  CborError rc; // CborNoError == 0
+
   // Parse message_span.
-  CborParser cbor_parser;
+  CborParser parser;
```

Comment:
why get rid of cbor_?

Reply:
why have it?  It shortened lines which plays better with clang-format.  If it's preferred I can put it back.